crates/router/src/core/fraud_check/flows/record_return.rs

```diff
@@ -71,7 +71,7 @@ impl ConstructFlowSpecificData<RecordReturn, FraudCheckRecordReturnData, FraudCh
                 amount: self.payment_attempt.amount,
                 refund_method: RefundMethod::OriginalPaymentInstrument, //we dont consume this data now in payments...hence hardcoded
                 currency,
-                refund_transaction_id: self.refund.clone().map(|refund| refund.refund_id),
+                refund_transaction_id: None,
```

Comment:
I was under impression that record_refund should be depricated as we are cancelling payment instead of refunding. But we can still have record_refund and needs to be invoked in refunds flow. I reverted this changes